packages/@aws-cdk/aws-config/test/integ.scoped-rule.ts

```diff
@@ -4,12 +4,15 @@ import * as config from '../lib';
 
 const app = new cdk.App();
 
-const stack = new cdk.Stack(app, 'aws-cdk-config-rule-scoped-integ');
+const stack = new cdk.Stack(app, 'aws-cdk-config-rule-scoped-integ', {
+  env: { account: '123456789012', region: 'us-east-1' },
```

Comment:
```suggestion
```

Reply:
Let me leave this comment as it is. It is certainly not a process that asks for region and account ID. However, if it is randomized, the hash seed of the snapshot test will change every time, and the test will not converge.